go/apps/api/routes/v2_identities_delete_identity/handler.go

```diff
@@ -68,9 +70,9 @@ func (h *Handler) Handle(ctx context.Context, s *zen.Session) error {
 		return err
 	}
 
-	identity, err := db.Query.FindIdentityByExternalID(ctx, h.DB.RO(), db.FindIdentityByExternalIDParams{
+	identity, err := db.Query.FindIdentityByID(ctx, h.DB.RO(), db.FindIdentityByIDParams{
 		WorkspaceID: auth.AuthorizedWorkspaceID,
-		ExternalID:  req.ExternalId,
+		Identity:    req.ExternalId,
```

Comment:
the openapi.V2IdentitiesDeleteIdentityRequestBody shouldn't have a field called `ExternalID`, no? it should now have an `Identity` field